packages/compass-data-modeling/src/services/export-diagram.tsx

```diff
@@ -155,6 +155,17 @@ export function getExportJsonFromModel({
   };
 }
 
+export function exportEdits(fileName: string, edits: Edit[]) {
+  const json = { name: fileName, edits };
+  const blob = new Blob([JSON.stringify(json, null, 2)], {
+    type: 'application/json',
+  });
```

Comment:
I think we discussed adding a version fiend here and maybe it's a good idea to completely obscure the content with something simple like base64-ing the the whole string? In the spirit of making it not easily parseable by users at all

Reply:
added in [1cfac05](https://github.com/mongodb-js/compass/pull/7116/commits/1cfac055a5e1a854c980271cf43a07a36fb7b2e2)